src/lib/profiles/data-management/Current/LoggingManagement.cpp

```diff
@@ -1817,8 +1732,7 @@ void LoggingManagement::FlushHandler(System::Layer * inSystemLayer, INET_ERROR i
 
         break;
     }
-    case kLoggingManagementState_Holdoff:
-    {
+    case kLoggingManagementState_Holdoff: {
```

Comment:
Style: new line before this line, consistent with line 1751.

Reply:
done